ncloud/resource_ncloud_login_key.go

```diff
@@ -80,96 +78,174 @@ func resourceNcloudLoginKeyUpdate(d *schema.ResourceData, meta interface{}) erro
 }
 
 func resourceNcloudLoginKeyCreate(d *schema.ResourceData, meta interface{}) error {
-	client := meta.(*ProviderConfig).Client
+	var privateKey *string
+	var err error
 
 	keyName := d.Get("key_name").(string)
-	reqParams := &server.CreateLoginKeyRequest{KeyName: ncloud.String(keyName)}
 
-	logCommonRequest("CreateLoginKey", reqParams)
+	if meta.(*ProviderConfig).SupportVPC == true || meta.(*ProviderConfig).Site == "fin" {
+		privateKey, err = createVpcLoginKey(meta.(*ProviderConfig), &keyName)
+	} else {
+		privateKey, err = createClassicLoginKey(meta.(*ProviderConfig), &keyName)
+	}
 
-	resp, err := client.server.V2Api.CreateLoginKey(reqParams)
 	if err != nil {
-		logErrorResponse("CreateLoginKey", err, keyName)
 		return err
 	}
-	logCommonResponse("CreateLoginKey", GetCommonResponse(resp))
 
 	d.SetId(keyName)
-	d.Set("private_key", resp.PrivateKey)
+	d.Set("private_key", privateKey)
```

Comment:
```suggestion
	d.Set("private_key", (*privateKey)[:len(*privateKey)-1])
```
위와 같이 변경하니 테스트코드에서  `ExpectNonEmptyPlan: true` 를 지워도 테스트 통과가 되긴 합니다.
맨 마지막 `\n` 이 `state` 설정에서 무시되어서, DIFF 로 인식 하려고 하는것 같네요.

```
...
 ---END RSA PRIVATE KEY-----\n
```

Reply:
`Statefunc` 에서 사용한 것과 동일하게 `Trim.space` 으로 처리했어요.